server/fishtest/userdb.py

```diff
@@ -92,6 +93,7 @@ def create_user(self, username, password, email):
                 }
             )
             self.last_pending_time = 0
+            clear_cache()
```

Comment:
is this needed?

Reply:
the function writes to the db, so it makes it safe. If we assume that create user does always create a new user, that new user can't be in the cache yet.

Reply:
since it actually checks, it is not needed.

Reply:
I was thinking that the new user will be added to the cache if/when it is searched for, no need to clear all the other users that are already in the cache

Reply:
right, just wanted to avoid the case where create_user would create an existing user with a new password, but that can't happen as the check is explicit.